app/components/ui/copy-on-launch-modal/component.js

```diff
@@ -31,7 +32,12 @@ export default Component.extend({
               let handleLaunchError = (tale, err) => {
                 console.error('Failed to launch Tale', err);
                 self.set("errorMessage", err.message);
-                $('.ui.modal.compose-error').modal('show');
+                $('.ui.modal.compose-error').modal({
```

Comment:
```suggestion
                $('#copy-error-modal').modal({
```

This should fix the duplicate modal issue for the copy-on-launch error case